website_blog_share/README.rst

```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+Blog Share
+=========
+
+This module brings the users the possibility to globally enable/disable a share button 
+that will be displayed in each blog post.
+
+Usage
+=====
+
+1. Go to website menu
+2. Open one of your blog post
+3. Click on customize button
+4. Toggle share blog option to enable/disable share button
+
+NB: even if you have to go to one specific blog post, the set up is valid for all of them.
+
+Know issues / Roadmap
+====================
+* Share product configuration valid for all products. No individual configuration for each product
```

Comment:
Is this **Known issues / Roadmap** section ok?,
Its seems to be a copy of the other PR you made before https://github.com/OCA/e-commerce/pull/98/files#diff-64255abdcd1527c8462793394b6af4c1R21, this is a description talking about products and this PR references to Blog posts.